VK_App/VK_App/ViewControllers/SecondBarVC/NewsTableViewController.swift

```diff
@@ -8,21 +8,96 @@ final class NewsTableViewController: UITableViewController {
     // MARK: Constants
 
     private enum Constants {
-        static let newsCellID = "newsCell"
+        static let emptyText = ""
+        static let headerCellID = "NewsHeaderCell"
+        static let textCellID = "NewsTextCell"
+        static let footerCellID = "NewsFooterCell"
+    }
+
+    private enum NewsCellType: Int, CaseIterable {
+        case header
+        case content
+        case footer
+    }
+
+    // MARK: Private properties
+
+    private let network = NetworkService()
```

Comment:
```suggestion
    private let networkService = NetworkService()
```
пора уже запомнить что если тип NetworkService то и его экземпляр networkService

Reply:
я помню, запамятовал поправить этот момент